sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -177,6 +178,21 @@ def tune(
                 values check this doc: https://www.kubeflow.org/docs/components/katib/experiment/#configuration-spec.
             parallel_trial_count: Number of Trials that Experiment runs in parallel.
             max_failed_trial_count: Maximum number of Trials allowed to fail.
+            resources_per_trial: A parameter that lets you specify how much
+            resources each trial container should have. You can either specify a
+            kubernetes.client.V1ResourceRequirements object (documented here:
+            https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1ResourceRequirements.md)
+            or a dictionary that includes one or more of the following keys:
+            `cpu`, `memory`, or `gpu` (other keys will be ignored). Appropriate
```

Comment:
@droctothorpe Can you mention that `gpu` means NVIDIA GPU, and if users want to use other GPUs, users must pass the values as a client.V1ResourceRequirements form?

Reply:
Good call. Thanks for the feedback. Done!

Reply:
@droctothorpe As I can see, the commit isn't pushed to this PR. Can you check if the commit is pushed?

Reply:
Doh! Forgot to `git add` 🙃 . Thanks, @tenzen-y! Fixed.